frontend/app/src/locales/en.json

```
@@ -350,6 +353,25 @@
     "terminate": "Terminate",
     "message": "rotki's minimum supported macOs version is Mojave(10.14)"
   },
+  "merge_dialog": {
+    "cancel": "Cancel",
+    "close": "Close",
+    "done": "The asset merge is complete. The old asset will still appear in the frontend until the next manual refresh or re-login.",
```

This sounds a bit like no action is needed. Perhaps use imperative form.
```suggestion
    "done": "The asset merge is complete. The old asset will still appear in the frontend. To make sure the new one is shown everywhere either restart the application or manually refresh balances in each place the old asset appears.",
```